src/openrct2/peep/Peep.cpp

```diff
@@ -758,147 +589,133 @@ static void peep_decide_whether_to_leave_park(rct_peep *peep)
 
 /** rct2: 0x009822F4, 0x00982310 */
 static const uint8 byte_9822F4[] = {
-    0,      // SHOP_ITEM_BALLOON
-    0,      // SHOP_ITEM_TOY
-    0,      // SHOP_ITEM_MAP
-    0,      // SHOP_ITEM_PHOTO
-    0,      // SHOP_ITEM_UMBRELLA
-    100,    // SHOP_ITEM_DRINK
-    150,    // SHOP_ITEM_BURGER
-    120,    // SHOP_ITEM_CHIPS
-    60,     // SHOP_ITEM_ICE_CREAM
-    50,     // SHOP_ITEM_CANDYFLOSS
-    0,      // SHOP_ITEM_EMPTY_CAN
-    0,      // SHOP_ITEM_RUBBISH
-    0,      // SHOP_ITEM_EMPTY_BURGER_BOX
-    150,    // SHOP_ITEM_PIZZA
-    0,      // SHOP_ITEM_VOUCHER
-    75,     // SHOP_ITEM_POPCORN
-    133,    // SHOP_ITEM_HOT_DOG
-    110,    // SHOP_ITEM_TENTACLE
-    0,      // SHOP_ITEM_HAT
-    50,     // SHOP_ITEM_CANDY_APPLE
-    0,      // SHOP_ITEM_TSHIRT
-    80,     // SHOP_ITEM_DONUT
-    90,     // SHOP_ITEM_COFFEE
-    0,      // SHOP_ITEM_EMPTY_CUP
-    170,    // SHOP_ITEM_CHICKEN
-    115,    // SHOP_ITEM_LEMONADE
-    0,      // SHOP_ITEM_EMPTY_BOX
-    0,      // SHOP_ITEM_EMPTY_BOTTLE
-    0xFF,
-    0xFF,
-    0xFF,
-    0xFF,
-    0,      // SHOP_ITEM_PHOTO2
-    0,      // SHOP_ITEM_PHOTO3
-    0,      // SHOP_ITEM_PHOTO4
-    70,     // SHOP_ITEM_PRETZEL
-    85,     // SHOP_ITEM_CHOCOLATE
-    95,     // SHOP_ITEM_ICED_TEA
-    90,     // SHOP_ITEM_FUNNEL_CAKE
-    0,      // SHOP_ITEM_SUNGLASSES
-    130,    // SHOP_ITEM_BEEF_NOODLES
-    120,    // SHOP_ITEM_FRIED_RICE_NOODLES
-    100,    // SHOP_ITEM_WONTON_SOUP
-    110,    // SHOP_ITEM_MEATBALL_SOUP
-    110,    // SHOP_ITEM_FRUIT_JUICE
-    90,     // SHOP_ITEM_SOYBEAN_MILK
-    100,    // SHOP_ITEM_SU_JONGKWA
-    130,    // SHOP_ITEM_SUB_SANDWICH
-    75,     // SHOP_ITEM_COOKIE
-    0,      // SHOP_ITEM_EMPTY_BOWL_RED
-    0,      // SHOP_ITEM_EMPTY_DRINK_CARTON
-    0,      // SHOP_ITEM_EMPTY_JUICE_CUP
-    115,    // SHOP_ITEM_ROAST_SAUSAGE
-    0       // SHOP_ITEM_EMPTY_BOWL_BLUE
+    0,   // SHOP_ITEM_BALLOON
+    0,   // SHOP_ITEM_TOY
+    0,   // SHOP_ITEM_MAP
+    0,   // SHOP_ITEM_PHOTO
+    0,   // SHOP_ITEM_UMBRELLA
+    100, // SHOP_ITEM_DRINK
+    150, // SHOP_ITEM_BURGER
+    120, // SHOP_ITEM_CHIPS
+    60,  // SHOP_ITEM_ICE_CREAM
+    50,  // SHOP_ITEM_CANDYFLOSS
+    0,   // SHOP_ITEM_EMPTY_CAN
+    0,   // SHOP_ITEM_RUBBISH
+    0,   // SHOP_ITEM_EMPTY_BURGER_BOX
+    150, // SHOP_ITEM_PIZZA
+    0,   // SHOP_ITEM_VOUCHER
+    75,  // SHOP_ITEM_POPCORN
+    133, // SHOP_ITEM_HOT_DOG
+    110, // SHOP_ITEM_TENTACLE
+    0,   // SHOP_ITEM_HAT
+    50,  // SHOP_ITEM_CANDY_APPLE
+    0,   // SHOP_ITEM_TSHIRT
+    80,  // SHOP_ITEM_DONUT
+    90,  // SHOP_ITEM_COFFEE
+    0,   // SHOP_ITEM_EMPTY_CUP
+    170, // SHOP_ITEM_CHICKEN
+    115, // SHOP_ITEM_LEMONADE
+    0,   // SHOP_ITEM_EMPTY_BOX
+    0,   // SHOP_ITEM_EMPTY_BOTTLE
+    0xFF, 0xFF, 0xFF, 0xFF,
```

Comment:
Either you do put a table on one line or every item on its own line. Do not mix it up.

Reply:
Still needs fixing